### ISSUE TITLE
TreeView images not scaled on high DPI

### ISSUE THREAD

**Issue by tomlevels** [dotnet/winforms#7868]
### .NET version

.NET 6.0.9

### Did it work in .NET Framework?

Yes

### Issue description

When a monitor is scaled, for example 150%, tree view images are still small while labels are scaled well. See also https://github.com/dotnet/winforms/issues/6380

I know the issue is solved in .NET 7 preview, but will this be backported to .NET 6 since version 6 is a LTS version which we plan to use for some time?

### Steps to reproduce

See https://github.com/dotnet/winforms/issues/6380

**Comment by RussKie** [comment]
@tomlevels is this an adoption blocker for you?

**Comment by merriemcgaw** [comment]
.NET 6 is not fully compatible with scaling yet, though we did make a ton of progress over the past couple of releases. Unfortunately some of the infrastructure we need to make this a safe change in .NET 6 servicing is not there and we can't bring that code back. My recommendation is to take your dependencies on .NET 8 where most problems should be resolved (including this, as you pointed out was fixed for .NET 7).
